docs/readthedocs/models/input/grid/gridcontainer.md

```diff
@@ -24,14 +24,12 @@ and their higher voltage coupling point.
 
 Let's shed a more detailed light on the boundaries of a sub grid as of our definition.
 This especially is important, if the switchgear of the transformer is modeled in detail.
-We defined, that all nodes in upstream direction of the transformer, that are connected by switches *only* (therefore
-are within the switchgear) are counted towards the inferior sub grid structure (here "2"), although they belong to a
-different voltage level.
-This decision is taken, because we assume, that the interest to operate on the given switchgear will most likely be
-placed in the inferior grid structure.
-
-The "real" coupling node A is not comprised in the sub grids node collection, but obviously has reference through the
-switch between nodes A and B.
+We defined, that all nodes in upstream direction of the transformer, including those, which are connected by switches *only*
+(therefore are within the switchgear) are counted towards the superior sub grid structure (here "1"), because they belong
```

Comment:
I guess this should be refactored. Minimum the *only* does not make sense any longer.